readme.md

```diff
@@ -357,8 +357,8 @@ but no Tracking from Microsoft.
 + VS Code: https://code.visualstudio.com as a backup or read-only viewer when we are working on multiple projects concurrently.
 + Sublime Text: [sublimetext.com](https://www.sublimetext.com) - fast and free-ish 
   (you will be reminded to purchase a license ... but the license is perpetual not annual!)
-  
-  
++ CodeLobster: [codelobster.com](https://codelobster.com) - free cross-platform editor with plug-ins for all popular frameworks 
```

Comment:
Looks retro 🤓 

<img width="1340" alt="image" src="https://github.com/dwyl/dev-setup/assets/194400/80a671b0-4b45-4a06-9efe-1fdb80363bc3">

![image](https://github.com/dwyl/dev-setup/assets/194400/cca96a83-c5f6-4305-ae2d-10b9bf6eef25)

_Very_ focussed on `PHP` ecosystem with strong emphasis on the up-sell for more features: https://www.codelobster.com/order-ide.php

<img width="1457" alt="image" src="https://github.com/dwyl/dev-setup/assets/194400/1afde659-d513-4e0b-8df3-46c35564e5e0">

Unclear where the source code is.